### ISSUE TITLE
Incorrect link to online demo in readme?

### ISSUE THREAD

**Issue by miksh7** [passwordless-lib/fido2-net-lib#407]
The readme link to online demo redirects to Bitwarden website

> Online` examples: https://www.passwordless.dev/

Should it go to https://fido2-net-lib.passwordless.dev/passwordless#heroFoot instead?

**Comment by abergs** [comment]
Yup, we should update that one. I might move it to a personal domain instead. Keeping this open as a reminder.

**Comment by Regenhardt** [comment]
Maybe ask bitwarden if they really want to keep the passwordless.dev name? Would be nice to have that .dev domain.

**Comment by Regenhardt** [comment]
Ah I see. Would be nice to have it on the passwordless.dev domain, even though that inadvertently links the library to bitwarden.

**Comment by My1** [comment]
> I'm employed by Bitwarden and work on building Passwordless.dev. Bitwarden will continue to use that domain. While Bitwarden do not own this library, they do help out with allowing me to work on it and other things needed for the library.

that explains some stuff when I looked at the site, I was like WTF why did bitwarden hijack this neat little project, which especially is one of the few ones that actually properly differentiates between passwordless and usernameless and not walking around calling resident creds just passwordless as if you cant do that normally too.

**Comment by abergs** [comment]
Readme has been edited and the odd link removed.
